docs/modules/user-guide/partials/proc_adding-plugin-component-to-a-devfile.adoc

```diff
@@ -14,74 +14,33 @@ This section describes how to add a `plugin` component to a devfile.
 
 . Define a component using the type `plugin`.
 
-. Define the `id` attribute. It is slash-separated publisher, name and version of plug-in from the Plug-in registry. List of available {prod-short} plug-ins and more information about registry can be found in the link:https://github.com/eclipse/che-plugin-registry[{prod-short} plug-in registry] GitHub repository.
+. Define the `id` attribute. It is the stack name in a devfile v2 registry. Devfile v2 registry should follow the convention according to the link:https://github.com/devfile/api/blob/master/docs/proposals/registry/index-server-REST-API.md[proposal].
 +
 .A devfile defining a plug-in `id`
 ====
 [source,yaml]
 ----
   components:
-   - name: exec-plugin
+   - name: plugin-name
      plugin:
-      id: machine-exec-plugin/0.0.1
+      id: nodejs
 ----
 ====
 
-. Optionally, specify an alternative component registry using the `registryUrl` parameter:
+. Optionally, specify an alternative devfile registry using the `registryUrl` parameter:
```

Comment:
IBM SG: Indicating optional and conditional steps
If a step is optional, begin the step with the word Optional. If your authoring tool provides a semantic element to generate the word Optional, use that element.
 1. Optional: Click Set password. In the Password field, type a password